plugins/action/dcnm_vrf.py

```diff
@@ -16,35 +16,1530 @@
 
 __metaclass__ = type
 
+# Standard Library Imports
+import copy
+import json
+import time
+import traceback
+from datetime import datetime
+
+# Ansible Imports
 from ansible_collections.ansible.netcommon.plugins.action.network import (
     ActionModule as ActionNetworkModule,
 )
 from ansible.utils.display import Display
+from ansible.errors import AnsibleError
+
+# Module Constants
+WAIT_TIME_FOR_DELETE_LOOP = 5
+VALID_VRF_STATES = ["DEPLOYED", "PENDING", "NA"]
+MAX_RETRY_COUNT = 50
 
 display = Display()
 
 
+class Logger:
```

Comment:
This `Logger` Class might be something that would be valuable to include in both the VRF and Network action plugins and call it from a shared location.  Can you sync with Neil on this?  Don't make the change yet but let's evaluate  first